media-sound/beets/beets-1.4.9-r1.ebuild

```diff
@@ -1,82 +1,101 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-MY_PV=${PV/_beta/-beta.}
-MY_P=${PN}-${MY_PV}
-
-PYTHON_COMPAT=( python3_6 )
+DISTUTILS_SINGLE_IMPL=1
+PYTHON_COMPAT=( python{2_7,3_{6,7,8}} )
```

Comment:
Please remove python2.7since it is deprecated. Also, please check, if python3.8 works, since dev-python/jellyfish has no python target for it.

Reply:
Hey @rain0r thank you for the review.

> Please remove python2.7 since it is deprecated.

I am not sure exactly, but python2.7 get a last release (2.7.18) on 2020-04-20,
so maybe we can drop the support after gentoo stop python2.7 itself ?

> Also, please check, if python3.8 works, since dev-python/jellyfish has no python target for it.

I will try but python3.8 is still new on my system and not easy to test.

Reply:
I have test several PYTHON_TARGETS, python3.7 and python3.8 they both don't work.

I think they some dependency, so I am not sure if it best to drop those support until all the dependency are fine or keep it like that @rain0r ?

Reply:
> Hey @rain0r thank you for the review.
> 
> > Please remove python2.7 since it is deprecated.
> 
> I am not sure exactly, but python2.7 get a last release (2.7.18) on 2020-04-20,
> so maybe we can drop the support after gentoo stop python2.7 itself ?
> 
> > Also, please check, if python3.8 works, since dev-python/jellyfish has no python target for it.
> 
> I will try but python3.8 is still new on my system and not easy to test.

Python 2 is officially retired and in the last months, many packages in portage have dropped python 2 support, so no need to support python 2 when writing a new ebuild.

Reply:
> I have test several PYTHON_TARGETS, python3.7 and python3.8 they both don't work.
> 
> I think they some dependency, so I am not sure if it best to drop those support until all the dependency are fine or keep it like that @rain0r ?

I'm using beets with python3.7 for some time now (got my own live ebuild). No problems with that.